src/selection/Selection.js

```diff
@@ -211,8 +222,14 @@ define(
                 context: context,
                 element: element
             };
+
+            if (!this.isSelectable(element)) {
+                return () => {};
+            }
```

Comment:
@davetsay Very good solution, thanks for the fix! I realize the PR is merged, but I have a minor comment. I would  check for isSelectable() first at the very beginning before declaring the selectable variable.

Reply:
@psarram , you're absolutely right. Fixed in #2535.